service/src/main/java/tds/assessment/web/endpoints/AssessmentController.java

```diff
@@ -9,36 +9,38 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import tds.assessment.SetOfAdminSubject;
-import tds.assessment.services.AdminSubjectService;
+import tds.assessment.Assessment;
+import tds.assessment.services.AssessmentService;
 import tds.common.web.exceptions.NotFoundException;
 
 /**
  * Controller handling admin subject related entities
  */
 @RestController
-@RequestMapping("/assessments/admin-subject")
-class AdminSubjectController {
-    private final AdminSubjectService service;
+@RequestMapping("/assessments/assessment")
```

Comment:
I think we should drop the `assessment`.  The only reason `admin-subject` was there is because I didn't understand that it represented an assessment.  Now I think the `key` is the id so `/assessments/key` makes sense to me.